### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by Kashif-Raza6** [PromtEngineer/localGPT#41]
How can we save the embeddings of a particular file and use it later for question-answering?

**Comment by PromtEngineer** [comment]
Good point, currently it store add the embeddings into the DB folder, will add this.
